test/hooks/useSafeLayoutEffect.spec.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import { render } from "test/testing-utils/renderer";
+import { useFetch } from "hooks/useFetch";
+import { isServerSide } from "utils/shared-utils";
+import ComponentUsingSafeLayout from "test/testing-utils/hook-components/usePolicyComponent";
+
+jest.mock("hooks/useFetch");
+jest.mock("utils/shared-utils");
+
+describe("useSafeLayoutEffect", () => {
+  let policy, fetchResponse;
+
+  beforeEach(() => {
+    policy = {
```

Comment:
Is this setup necessary? Same for lines 45/46

Reply:
There are parts of this set-up needed because I'm using a component for another hook that consumes this one to test the logic instead of rewriting a new component for this.  I'll remove the extra stuff though.